libpod/container_internal.go

```diff
@@ -914,8 +914,10 @@ func (c *Container) makeBindMounts() error {
 		c.state.BindMounts = make(map[string]string)
 	}
 
-	// SHM is always added when we mount the container
-	c.state.BindMounts["/dev/shm"] = c.config.ShmDir
+	if c.config.ShmDir != "none" {
```

Comment:
Instead of this - we should probably check the spec for an IPC namespace, I think

Reply:
I am not sure what you mean?

Reply:
The namespaces the container is using are listed in the OCI spec. If there is no IPC namespace listed, that means that we have an IPC=host container, so we don't need to pass in a special keyword to tell libpod not to make an SHM mount